src/screens/QRScannerScreenWithData.js

```diff
@@ -78,7 +78,9 @@ class QRScannerScreenWithData extends Component {
 
     if (!data) return null;
     this.setState({ enableScanning: false });
-    Vibration.vibrate();
+    if (!__DEV__) {
```

Comment:
why?

Reply:
Because `Vibration.vibrate()` crushes the simulator

Reply:
can you use 
`isSimulator = __DEV__ && (await DeviceInfo.isEmulator());` ?

Reply:
Didn't know about it! Thanks for sharing!